src/server/common/components/entity/template.njk

```diff
@@ -74,6 +75,12 @@ data-testid="app-entity{% if params.index %}-{{ params.index }}{% endif %}">
         {% endfor %}
       </ul>
 
+    {% elseif params.kind === "button" %}
+      <form action="{{ params.url }}" method="POST">
+        <input type="hidden" name="csrfToken" value={{csrfToken}}>
+        {{ appButton({text: params.value, classes: "govuk-button--secondary"}) }}
```

Comment:
make `govuk-button--secondary` - `app-button--secondary` and you get our secondary button colour thats used all over the app

Reply:
oh actually your using the wrong button here. Our button has a loader on it

Reply:
just use

```
  {{ govukButton({
    classes: "app-button app-button--secondary",
    text: params.value
  }) }}
```

Reply:
done